_ext/eclipse-cdt/build.gradle

```diff
@@ -4,9 +4,12 @@ ext {
 	]
 
 	p2Repository = "https://download.eclipse.org/tools/cdt/releases/${VER_ECLIPSE_CDT}"
+	p2Repository = "file:////home/pinky/Downloads/cbi/eclipse-cdt/final/"
```

Comment:
@fvgh I'm a bit confused by this new P2 repository. Could you help me understand why it's included? :)

Reply:
Good catch!  I'll remove it real quick :)